src/bci_build/package/rust/README.md.j2

```diff
@@ -0,0 +1,48 @@
+# {{ image.pretty_name }} Container Image
+
+{% include 'badges.j2' %}
+
+## Description
+
+[Rust](https://www.rust-lang.org/) is a systems programming language sponsored by Mozilla Research. It is designed to be a "safe, concurrent, practical language", supporting functional and imperative-procedural paradigms. Rust is syntactically similar to C++, but is designed for better memory safety while maintaining performance.
```

Comment:
```suggestion
[Rust](https://www.rust-lang.org/) is a systems programming language sponsored by Mozilla Research. It is designed to be a "safe, concurrent, practical language", supporting functional and imperative-procedural paradigms.  While syntactically similar to C++, Rust is designed for better memory safety without performance penalty.
```

Why "safe, concurrent, practical language" is in quotes? Quotes make it sound like it's safe but not really.

Reply:
copy-paste from the official Rust image, we can remove that, it is indeed weird and I did not notice it.

Reply:
Fixed.